<launch>
	<!-- urc_gui_common RosLink args -->
	<arg name="local_position_topic" default="/local_position"/>
	<arg name="global_position_topic" default="/global_position"/>
	<arg name="global_origin_topic" default="/global_origin"/>
	<arg name="marker_list_topic" default="/marker_list"/>
	<arg name="state_topic" default="/state"/>
	<arg name="planner_status_topic" default="/planner_status"/>
	<arg name="select_planner_service" default="/select_planner_service"/>
	<arg name="pid_planner_name" default="pid_planner"/>
	<arg name="dwa_planner_name" default="dwa_planner"/>
	<arg name="pid_planner_enabled_service" default="/pid_planner_enabled"/>
	<arg name="dwa_planner_enabled_service" default="/dwa_planner_enabled"/>
	<arg name="pid_planner_status_topic" default="/pid_planner_status"/>
	<arg name="dwa_planner_status_topic" default="/dwa_planner_status"/>
	<arg name="found_marker_list_topic" default="/found_marker_list"/>
	<arg name="add_marker_service" default="/add_marker"/>
	<arg name="clear_markers_service" default="/clear_markers"/>
	<arg name="reorder_marker_service" default="/reorder_marker"/>
	<arg name="reorder_markers_service" default="/reorder_markers"/>
	<arg name="edit_marker_service" default="/edit_marker"/>
	<arg name="remove_marker_service" default="/remove_marker"/>
	<arg name="insert_marker_service" default="/insert_marker"/>
	<arg name="drive_forward_service" default="/drive_forward"/>
	<arg name="car_style_turning_service" default="/car_style_turning"/>
	<arg name="clear_found_markers_service" default="/clear_found_markers"/>

	<!-- rover_gui_common RosLink args -->
	<arg name="ed_waypoint_list_topic" default="/ed_waypoint_list"/>

	<!-- launch the gui -->
	<node pkg="wanderer_gui" name="$(anon wanderer_gui)"
	type="main.py" output="screen" respawn="false">
		<param name="local_position_topic" value="$(arg local_position_topic)"/>
		<param name="global_position_topic" value="$(arg global_position_topic)"/>
		<param name="global_origin_topic" value="$(arg global_origin_topic)"/>
		<param name="marker_list_topic" value="$(arg marker_list_topic)"/>
		<param name="state_topic" value="$(arg state_topic)"/>
		<param name="planner_status_topic" value="$(arg planner_status_topic)"/>
		<param name="select_planner_service" value="$(arg select_planner_service)"/>
		<param name="pid_planner_name" value="$(arg pid_planner_name)"/>
		<param name="dwa_planner_name" value="$(arg dwa_planner_name)"/>
		<param name="pid_planner_enabled_service" value="$(arg pid_planner_enabled_service)"/>
		<param name="dwa_planner_enabled_service" value="$(arg dwa_planner_enabled_service)"/>
		<param name="pid_planner_status_topic" value="$(arg pid_planner_status_topic)"/>
		<param name="dwa_planner_status_topic" value="$(arg dwa_planner_status_topic)"/>
		<param name="found_marker_list_topic" value="$(arg found_marker_list_topic)"/>
		<param name="add_marker_service" value="$(arg add_marker_service)"/>
		<param name="clear_markers_service" value="$(arg clear_markers_service)"/>
		<param name="reorder_marker_service" value="$(arg reorder_marker_service)"/>
		<param name="reorder_markers_service" value="$(arg reorder_markers_service)"/>
		<param name="edit_marker_service" value="$(arg edit_marker_service)"/>
		<param name="remove_marker_service" value="$(arg remove_marker_service)"/>
		<param name="insert_marker_service" value="$(arg insert_marker_service)"/>
		<param name="drive_forward_service" value="$(arg drive_forward_service)"/>
		<param name="car_style_turning_service" value="$(arg car_style_turning_service)"/>
		<param name="clear_found_markers_service" value="$(arg clear_found_markers_service)"/>

		<param name="ed_waypoint_list_topic" value="$(arg ed_waypoint_list_topic)"/>
	</node>
</launch>
